### DISCUSSION TITLE
useTheme() with external style file

### DISCUSSION THREAD

**Discussion by ajo-alex** [callstack/react-native-paper#3252] [Q&A]
I have a functional component ***(Card.tsx)*** and an external style file ***(Card.style.ts)***

I can import my colors by using useTheme() in my component.
How to import these colors in my style file.

**Comment by ArnabXD** [comment]
You can use custom theme objects or the provided themes (DefaultTheme, DarkTheme) in the stylesheet, but you can't include them in the stylesheet conditionally for the dark and light themes. It would be best if you wrote the condition inside the component.
